packages/flutter/lib/src/gestures/events.dart

```diff
@@ -790,6 +790,61 @@ class PointerUpEvent extends PointerEvent {
   );
 }
 
+/// A base type for events that correspond to a discrete pointer signal.
+///
+/// There are events that originate from the pointer but don't change the state
+/// of the pointer itself, and are discrete rather than needing to be
+/// interpreted in the context of a series of events.
+abstract class PointerSignalEvent extends PointerEvent {
+  /// Abstract const constructor. This constructor enables subclasses to provide
+  /// const constructors so that they can be used in const expressions.
+  const PointerSignalEvent({
+    Duration timeStamp = Duration.zero,
+    int pointer = 0,
+    PointerDeviceKind kind = PointerDeviceKind.mouse,
+    int device = 0,
+    Offset position = Offset.zero,
+  }) : super(
+    timeStamp: timeStamp,
+    pointer: pointer,
+    kind: kind,
+    device: device,
+    position: position,
+  );
+}
+
+/// The pointer issued a scroll event.
```

Comment:
I'd word this differently: e.g. "A pointer event object that contains information about pointer scroll events."

Reply:
Hrm.  I'd actually like to have them all be changed, since it's kind of crappy documentation.

But, in the interest of landing this, can you just file an issue to do so?  You can keep this consistent with the rest.